core/logx/writer.go

```diff
@@ -269,7 +269,11 @@ func combineGlobalFields(fields []LogField) []LogField {
 		return fields
 	}
 
-	return append(globals.([]LogField), fields...)
```

Comment:
Ignoring the order, maybe this is ok? `append(fields, globals.([]LogField)...)`

Reply:
Fields are not order sensitive. It's ok.

Reply:
done.